docs/LinuxTool/tree.md

```diff
@@ -2,8 +2,140 @@
 
 ## 1. 基本功用
 
+tree是linux系统下十分常用的一个小工具。其基本作用是用于以树状图的方式显示指定目录下存在的各级子目录和子文件。只使用linux内建的ls指令虽然能够获取任意目录下子目录和子文件的信息，但是却不能够展示完整地展示多层次的目录、文件及之间的关系。在阅览较为复杂的工程时，tree的功能就显得非常重要了,能够帮助我们对整个工程各部分的文件的关系有直观清晰的认识。
+
 ## 2. 安装方法
 
+- 1.确认系统的Linux版本
+- 请先确认所在的linux系统的版本是RedHat系列（包括RedHat,CentOS等）还是Debian系列（包括Debian,Ubuntu等）
+- 如果不是很清楚，请打开系统命令行输入指令'lsb\_release -a'，根据打印出的信息进行确认。确认完之后前往下一步
+- 2.输入指令安装tree
+- 如果是RedHat系列，请输入'sudo yum install tree';如果是Debian系列，请输入'sudo apt-get install tree'
+- 输入之后按下回车键，前往下一步。（如果不关心安装细节，可以在以上指令中加入'-y'参数，不需要重复确认地进行安装)
+- 3.进行安装
+- 由于调用了root权限，请按照提示输入密码并按回车键确认，之后根据软件安装中的提示输入y/n完成安装
+
 ## 3. 使用方法（基本参数和基本指令）
 
+tree的使用只需要按以下格式输入指令即可：
+
+tree[参数][目录]
+
+其中目录如果不输入的话默认为当前所在的工作目录
+
+tree的常用参数如下：
```

Comment:
This should be streesed so that I can easily find this section

Reply:
got it!